microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Build Microsoft Edge sidebar PWAs
+description: How to build your PWA for installation in the Microsoft Edge Sidebar.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: pwa
+ms.date: 02/22/2023
+---
+# Build Microsoft Edge sidebar PWAs
+
+![Experimental feature](../../media/experimental-tag.png)
+
+Progressive Web Apps (PWAs) can opt-in to be pinned to the sidebar in Microsoft Edge.
+
+The sidebar in Microsoft Edge allows users to easily access popular websites and utilities alongside their browser tabs. The content in the sidebar augments the user's primary task by enabling side-by-side browsing and minimizing the need to switch contexts between browser tabs.
+
+By signaling intent to be pinned to the sidebar, your PWA gets the following benefits:
+
+* Your PWA is promoted in the Microsoft Edge sidebar store.
```

Comment:
.. may be promoted. I imagine that eventually if we succeed, we will need to balance promotion with other recommendation criteria, e.g. user engagement with sites, etc.